templates.json

```diff
@@ -136,6 +136,28 @@
             "debugSSHPort": "2250",
             "multiContainer": false,
             "support": "Community"
+        },
+        {
+            "img": "python3console.png",
+            "folder": "python3Console",
+            "description": "Python 3 Console Application",
+            "runtime": "Python 3",
+            "language": "Python 3",
+            "debugPort": "6512",
+            "debugSSHPort": "6502",
+            "multiContainer": false,
+            "support": "Toradex"
+        },
+        {
+            "img": "python3pyside2.png",
+            "folder": "python3Pyside2QML",
+            "description": "Python 3 Pyside 2(Qt 5) QML Application",
+            "runtime": "Pyside 2",
+            "language": "Python 3",
+            "debugPort": "6512",
+            "debugSSHPort": "6502",
+            "multiContainer": false,
+            "support": "Community"
```

Comment:
The Python with Qt that will be supported by Toradex will be a PyQt6 template, which uses Qt6 instead of Qt5. However, PyQt6 is available just on the newest version of Debian, Debian Bookworm. So the Python 3 PqQt6 template will be available when the Torizon base containers using Debian Bookworm are available.